src/test/java/com/deque/html/axecore/selenium/AxeIntegrationUnitTest.java

```diff
@@ -145,4 +238,44 @@ private void initDriver(String browser) {
     webDriver.manage().timeouts().setScriptTimeout(20, TimeUnit.SECONDS);
     webDriver.manage().window().maximize();
   }
+
+  private String createReportPath() {
+    // TODO: generate ideal path to place report for testing

```

Comment:
not sure where I was supposed to generate a file path to for these unit tests. Let me know if a different direction is needed.